compute/api/Compute.Samples/CreateInstanceAsync.cs

```diff
@@ -22,6 +22,7 @@
 public class CreateInstanceAsyncSample
 {
     public async Task CreateInstanceAsync(
+        // TODO(developer): Replace these variables before running the sample.
```

Comment:
This text is not C# idiomatic nor it matches what this code is. It should read something like (Here and everywhere):
```suggestion
        // TODO(developer): Set your own default values for these parameters or pass different values when calling this method.
```